viewing_party/party.py

```diff
@@ -1,23 +1,112 @@
 # ------------- WAVE 1 --------------------
-
 def create_movie(title, genre, rating):
-    pass
+    movie_dict = {}
+    movie_dict["title"] = title
+    movie_dict["genre"] = genre
+    movie_dict["rating"] = rating
+    for value in movie_dict.values():
+        if value == None:

```

Comment:
Nice use of a loop to check for None values! This check works, though it is better practice to use `is` to check whether a variable is None. [This article](https://www.pythontutorial.net/advanced-python/python-none/) gives a good explanation.